hoodie-client/src/main/java/com/uber/hoodie/io/HoodieInsertHandle.java

```diff
@@ -129,6 +131,7 @@ public WriteStatus close() {
             stat.setPrevCommit(HoodieWriteStat.NULL_COMMIT);
             stat.setFileId(status.getFileId());
             stat.setFullPath(path.toString());
+            stat.setRelativePath(relativePath.toString());
```

Comment:
I think what we can do here, is to simply ONLY store the relative path instead of the full path and change wherever its expected as a full path.. The reason is we always know the basepath of the dataset.